securedrop/journalist_templates/_source_row.html

```diff
@@ -1,44 +1,58 @@
 {% set docs = source.documents_messages_count()['documents'] %}
 {% set msgs = source.documents_messages_count()['messages'] %}
-<li class="source {% if source.num_unread != 0 %}unread{% else %}read{% endif %}" data-source-designation="{{ source.journalist_designation|lower }}">
-  <time class="date" title="{{ source.last_updated|rel_datetime_format }}" datetime="{{ source.last_updated|html_datetime_format }}">{{ source.last_updated|rel_datetime_format(relative=True) }}</time>
-  <div class="designation">
+<tr class="source {% if source.num_unread != 0 %}unread{% else %}read{% endif %}" data-source-designation="{{ source.journalist_designation|lower }}">
+  <td aria-label="{{ source.last_updated|rel_datetime_format(relative=True) }}"><time class="date" title="{{ source.last_updated|rel_datetime_format }}" datetime="{{ source.last_updated|html_datetime_format }}">{{ source.last_updated|rel_datetime_format(relative=True) }}</time></td>
```

Comment:
I agree in theory!  In practice, at least my screen-reader (VoiceOver on macOS) reads the `datetime` attribute instead of the inner HTML, unless overridden with this `aria-label`.